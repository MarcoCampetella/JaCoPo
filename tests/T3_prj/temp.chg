 C     0.000000    5.142111    0.130773   -0.047530
 C     0.000000    4.619111    1.391633    0.021934
 H     0.000000    5.227910    2.295844   -0.019030
 C     0.000000    3.196401    1.391927   -0.065530
 H     0.000000    2.589532    2.298106   -0.032476
 C     0.000000    2.655142    0.128524    0.063625
 S     0.000000    3.911631   -1.070006   -0.059161
 C     0.000000    1.253330   -0.259361   -0.079000
 C     0.000000    0.708967   -1.520510    0.065166
 H     0.000000    1.312950   -2.428428    0.012034
 C     0.000000   -0.708967   -1.520510   -0.065166
 H     0.000000   -1.312950   -2.428428   -0.012034
 C     0.000000   -1.253330   -0.259361    0.079000
 S     0.000000    0.000000    0.942292    0.000000
 C     0.000000   -2.655142    0.128524   -0.063625
 C     0.000000   -3.196401    1.391927    0.065530
 H     0.000000   -2.589532    2.298106    0.032476
 C     0.000000   -4.619111    1.391633   -0.021934
 H     0.000000   -5.227910    2.295844    0.019030
 C     0.000000   -5.142111    0.130773    0.047530
 H     0.000000   -6.190497   -0.161680    0.027582
 S     0.000000   -3.911631   -1.070006    0.059161
 H     0.000000    6.190497   -0.161680   -0.027582
