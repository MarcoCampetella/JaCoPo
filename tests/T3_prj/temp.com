%nprocshared=2
%mem=500MB
%chk=temp.chk
#p td=(nstates=1) cam-b3lyp nosymm density=transition=1 iop(9/40=5) svp

Title

0 1
 C                  0.00000000    5.14211100    0.13077303
 C                  0.00000000    4.61911100    1.39163303
 H                  0.00000000    5.22791000    2.29584403
 C                  0.00000000    3.19640100    1.39192703
 H                  0.00000000    2.58953200    2.29810603
 C                  0.00000000    2.65514200    0.12852403
 S                  0.00000000    3.91163100   -1.07000597
 C                  0.00000000    1.25333000   -0.25936097
 C                  0.00000000    0.70896700   -1.52050997
 H                  0.00000000    1.31295000   -2.42842797
 C                  0.00000000   -0.70896700   -1.52050997
 H                  0.00000000   -1.31295000   -2.42842797
 C                  0.00000000   -1.25333000   -0.25936097
 S                  0.00000000    0.00000000    0.94229203
 C                  0.00000000   -2.65514200    0.12852403
 C                  0.00000000   -3.19640100    1.39192703
 H                  0.00000000   -2.58953200    2.29810603
 C                  0.00000000   -4.61911100    1.39163303
 H                  0.00000000   -5.22791000    2.29584403
 C                  0.00000000   -5.14211100    0.13077303
 H                  0.00000000   -6.19049700   -0.16167997
 S                  0.00000000   -3.91163100   -1.07000597
 H                 -0.00000000    6.19049700   -0.16167997

