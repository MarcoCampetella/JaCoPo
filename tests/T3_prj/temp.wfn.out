 Warning: "settings.ini" was found neither in current folder nor in the path defined by "Multiwfnpath" environment variable. Now using default settings instead
 
 Multiwfn -- A Multifunctional Wavefunction Analyzer (for Linux)
 Version 3.3.8, release date: 2015-Dec-1
 Project leader: Tian Lu  (Beijing Kein Research Center for Natural Sciences)
 Citation of Multiwfn: Tian Lu, Feiwu Chen, J. Comp. Chem. 33, 580-592 (2012)
 Multiwfn official website: http://Multiwfn.codeplex.com
 Multiwfn official forum (in Chinese): http://bbs.keinsci.com
 Bug reporting, question and suggestion, please contact: Sobereva@sina.com
 ( The number of threads:  4   Current date: 2016-03-03   Time: 18:27:16 )
 
 Input file path, for example E:\Haiyore!Nyaruko-san\Nyaruko.wfn
 (Supported types: .wfn/.wfx/.fch/.31/.chg/.pdb/.xyz/.cub/.grd/.molden, etc.)
 Hint: To reload the file last time used, simply input the letter "o". Input such as ?miku.fch can open miku.fch in the same folder of the file last time used
 Please wait...
 Loading various information of the wavefunction
 Loading basis-set definition...
 Loading orbitals...
 Converting basis function information to GTF information...
 Generating overlap matrix...
 Converting basis function information from Cartesian to spherical type...
 Loading post-HF density matrix...
 Note: Post-HF density matrix is loaded from .fch file, all of the following analyses based on basis information will be for post-HF wavefunction
 
 Total/Alpha/Beta electrons:    128.0000     64.0000     64.0000
 Net charge:     0.00000      Expected multiplicity:    1
 Atoms:     23,  Basis functions:    262,  GTFs:    467
 Total energy: -1655.555944966515 Hartree,   Virial ratio:  2.00467974
 This is a restricted post-HF wavefunction
 Title line of this file: Title
 Formula: H8 C12 S3 
 Molecule weight:       248.38692

 Loaded /home/dpadula/T3_prj/temp.fchk successfully!
 
                    ------------ Main function menu ------------ 
 -4 Exclude some atoms contribution to property
 -3 Obtain a fragment contribution to property
 0 Show molecular structure and view orbitals
 1 Output all properties at a point
 2 Topology analysis
 3 Output and plot specific property in a line
 4 Output and plot specific property in a plane
 5 Output and plot specific property within a spatial region
 6 Check & modify wavefunction
 7 Population analysis
 8 Orbital composition analysis
 9 Bond order analysis
 10 Plot Total/Partial/Overlap population density-of-states (DOS)
 11 Plot IR/Raman/UV-Vis/ECD/VCD spectrum
 12 Quantitative analysis of molecular surface
 13 Process grid data (No grid data is presented currently)
 14 Adaptive natural density partitioning (AdNDP) analysis
 15 Fuzzy atomic space analysis
 16 Charge decomposition analysis (CDA) and extended CDA (ECDA)
 17 Basin analysis
 18 Electron excitation analysis
 100 Other functions (Part1)
 200 Other functions (Part2)
             ------------ Electron excitation analyses ------------ 
 0 Return
 1 Analyze and visualize hole-electron distribution, transition dipole moment and transition density
 2 Plot transition density matrix as color-filled map
 3 Analyze charge-transfer based on density difference grid data (JCTC,7,2498)
 4 Calculate delta_r index to measure charge-transfer length (JCTC,9,3118)
 5 Calculate transition dipole moments between all excited states
 Input the path of the Gaussian output file or plain text file containing excitation data, e.g. c:\a.out
 There are    1 transition modes, analyze which one?  e.g. 2
 Multiplicity of this excited state is   1
 Excitation energy is   3.6774000 eV
 There are    4277 orbital pairs in this transition mode
 The sum of square of excitation coefficients:  0.505499
 The negative of the sum of square of de-excitation coefficients: -0.005504
 The sum of above two values  0.499995
 
 -1 Toggle calculating transit. magnetic dip. density in option 1, current: No
 0 Return
 1 Visualize and analyze hole, electron and transition density and so on
 2 Show contribution of MO pairs to transition dipole moment
 3 Show contribution of each MO to hole and electron distribution
 4 Generate and export transition density matrix
 5 Decompose transition dipole moment to basis function and atom contributions
 6 Calculate Mulliken atomic transition charges
 10 Modify or check excitation coefficients
 
 Please select a method to set up grid
 -10 Set extension distance of grid range for mode 1~4, current:  6.000000 Bohr
 1 Low quality grid   , covering whole system, about 125000 points in total
 2 Medium quality grid, covering whole system, about 512000 points in total
 3 High quality grid  , covering whole system, about 1728000 points in total
 4 Input the number of points or grid spacing in X,Y,Z, covering whole system
 5 Input original point, translation vector and the number of points
 6 Input center coordinate, number of points and extension distance
 7 The same as 6, but input two atoms, the midpoint will be defined as center
 8 Use grid setting of another cube file
Note: Don't view isosurface after generating the grid, the graph will be total incorrect
 Coordinate of origin in X,Y,Z is   -6.000000  -17.698344  -10.589064 Bohr
 Coordinate of end point in X,Y,Z is    6.015592   17.519772   10.541806 Bohr
 Grid spacing in X,Y,Z is    0.414331    0.414331    0.414331 Bohr
 The number of points in X,Y,Z is   30   86   52   Total:      134160
 Note: During the calculation of cross term of electron, the orbital pairs whose absolute value of coefficient <0.01 will be ignored to save time
 Calculating grid data...
 Finished:    1/   52
 Finished:    2/   52
 Finished:    3/   52
 Finished:    4/   52
 Finished:    5/   52
 Finished:    6/   52
 Finished:    7/   52
 Finished:    8/   52
 Finished:    9/   52
 Finished:   10/   52
 Finished:   11/   52
 Finished:   12/   52
 Finished:   13/   52
 Finished:   14/   52
 Finished:   15/   52
 Finished:   16/   52
 Finished:   17/   52
 Finished:   18/   52
 Finished:   19/   52
 Finished:   20/   52
 Finished:   21/   52
 Finished:   22/   52
 Finished:   23/   52
 Finished:   24/   52
 Finished:   25/   52
 Finished:   26/   52
 Finished:   27/   52
 Finished:   28/   52
 Finished:   29/   52
 Finished:   30/   52
 Finished:   31/   52
 Finished:   32/   52
 Finished:   33/   52
 Finished:   34/   52
 Finished:   35/   52
 Finished:   36/   52
 Finished:   37/   52
 Finished:   38/   52
 Finished:   40/   52
 Finished:   39/   52
 Finished:   41/   52
 Finished:   42/   52
 Finished:   43/   52
 Finished:   44/   52
 Finished:   45/   52
 Finished:   46/   52
 Finished:   47/   52
 Finished:   48/   52
 Finished:   49/   52
 Finished:   50/   52
 Finished:   51/   52
 Finished:   52/   52
 Totally took up CPU time       26.53s, wall clock time         7s

 Integral of hole:    1.002189
 Integral of electron:    1.004611
 Integral of transition density:    0.000283
 Transition dipole moment in X/Y/Z:  -0.000008   3.025226   0.001235 a.u.
 Integral of overlap of hole-electron distribution:   0.6136156
 Centroid of hole in X/Y/Z:        0.000057   -0.001024   -0.062190 Angstrom
 Centroid of electron in X/Y/Z:    0.000070   -0.003012    0.022741 Angstrom
 Distance between centroid of hole and electron in X/Y/Z:
    0.000013    0.001989    0.084932 Angstrom   Norm:    0.084955 Angstrom
 Variation of dipole moment with respect to ground state in X/Y/Z:
   -0.000024    0.003771   -0.161043 a.u.   Norm:    0.161087 a.u.
 RMSD of electron in x,y,z:   0.798   2.940   1.195   Total:   3.272 Angstrom
 RMSD of hole in x,y,z:       0.763   3.071   0.994   Total:   3.316 Angstrom
 H index in x,y,z:   0.780   3.005   1.094   Norm:   3.292 Angstrom
 t index in x,y,z:  -0.780  -3.003  -1.009   Norm:   3.263 Angstrom
 
 0 Return
 1 Show isosurface of hole distribution
 2 Show isosurface of electron distribution
 3 Show isosurface of hole and electron distribution simultaneously
 4 Show isosurface of overlap of hole-electron
 5 Show isosurface of transition density
 6 Show isosurface of transition dipole moment density
 7 Show isosurface of charge density difference
 8 Show isosurface of Cele and Chole functions simultaneously
 10 Output cube file of hole distribution to current folder
 11 Output cube file of electron distribution to current folder
 12 Output cube file of overlap of hole-electron to current folder
 13 Output cube file of transition density to current folder
 14 Output cube file of transition dipole moment density to current folder
 15 Output cube file of charge density difference to current folder
 16 Output cube file of Cele and Chole functions to current folder
 17 Output cube file of transition magnetic dipole moment density
 Outputting transition density to transdens.cub in current folder
 Please wait...
 Done!
 
 0 Return
 1 Show isosurface of hole distribution
 2 Show isosurface of electron distribution
 3 Show isosurface of hole and electron distribution simultaneously
 4 Show isosurface of overlap of hole-electron
 5 Show isosurface of transition density
 6 Show isosurface of transition dipole moment density
 7 Show isosurface of charge density difference
 8 Show isosurface of Cele and Chole functions simultaneously
 10 Output cube file of hole distribution to current folder
 11 Output cube file of electron distribution to current folder
 12 Output cube file of overlap of hole-electron to current folder
 13 Output cube file of transition density to current folder
 14 Output cube file of transition dipole moment density to current folder
 15 Output cube file of charge density difference to current folder
 16 Output cube file of Cele and Chole functions to current folder
 17 Output cube file of transition magnetic dipole moment density
 
 -1 Toggle calculating transit. magnetic dip. density in option 1, current: No
 0 Return
 1 Visualize and analyze hole, electron and transition density and so on
 2 Show contribution of MO pairs to transition dipole moment
 3 Show contribution of each MO to hole and electron distribution
 4 Generate and export transition density matrix
 5 Decompose transition dipole moment to basis function and atom contributions
 6 Calculate Mulliken atomic transition charges
 10 Modify or check excitation coefficients
 Generating transition density matrix...
 Done! Mulliken atomic transition charges have been outputted to atmtrchg.chg in current folder
 
 -1 Toggle calculating transit. magnetic dip. density in option 1, current: No
 0 Return
 1 Visualize and analyze hole, electron and transition density and so on
 2 Show contribution of MO pairs to transition dipole moment
 3 Show contribution of each MO to hole and electron distribution
 4 Generate and export transition density matrix
 5 Decompose transition dipole moment to basis function and atom contributions
 6 Calculate Mulliken atomic transition charges
 10 Modify or check excitation coefficients
